packages/firestore/test/integration/api/query.test.ts

```diff
@@ -61,6 +61,21 @@ apiDescribe('Queries', (persistence: boolean) => {
     });
   });
 
+  it('cannot issue limitToLast queries without explicit order-by', () => {
+    const testDocs = {
+      a: { k: 'a' },
+      b: { k: 'b' },
+      c: { k: 'c' }
+    };
+    return withTestCollection(persistence, testDocs, async collection => {
```

Comment:
I suspect you don't need `testDocs` here.

Reply:
Removed.